cmd/api/src/api/registration/v2.go

```diff
@@ -164,6 +164,7 @@ func NewV2API(cfg config.Configuration, resources v2.Resources, routerInst *rout
 		routerInst.GET("/api/v2/saved-queries", resources.ListSavedQueries).RequirePermissions(permissions.SavedQueriesRead),
 		routerInst.POST("/api/v2/saved-queries", resources.CreateSavedQuery).RequirePermissions(permissions.SavedQueriesWrite),
 		routerInst.PUT(fmt.Sprintf("/api/v2/saved-queries/{%s}", api.URIPathVariableSavedQueryID), resources.UpdateSavedQuery).RequirePermissions(permissions.SavedQueriesWrite),
+		routerInst.PUT(fmt.Sprintf("/api/v2/saved-queries/{%s}/unshare", api.URIPathVariableSavedQueryID), resources.UnshareSavedQuery).RequirePermissions(permissions.SavedQueriesWrite),
```

Comment:
Would a DELETE method on the `/share` endpoint be more apt? 🤔

Reply:
That could work.
I was debating between DELETE and PUT on the current endpoint. This endpoint is _only_ deleting entries, but it's a modification to the `saved-queries` which is the route that it's attached to.

In hindsight it may have been good to breakaway from our hackathon URL scheme and put these endpoints on their own resource IE:
`/api/v2/saved-queries/{%s}/permissions/share`

Reply:
Went ahead and changed the endpoint to be `DELETE api/v2/saved-queries/{%s}/permissions`

Also updated @ALCooper12 on the change to reflect the new routing in her PR